Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/API/UI/Driver.hs

```diff
@@ -91,6 +91,10 @@ type API =
                :> Post '[JSON] APISuccess
              :<|> "profile"
                :> ( TokenAuth
+                      :> Header "device" Text
```

Comment:
Lets create a separate API to update all this metadata of a driver and store it in table called DriverMetadata.